ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -1857,6 +1862,33 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe-mgmt
     workflow: cucushift-installer-rehearse-baremetalds-assisted-hypershift-agent-mce-disconnected
+- as: baremetalds-hypershift-agent-mce-n2minor-guest-f28
+  cluster: build05
```

Comment:
Perhaps use [capabilities](https://docs.ci.openshift.org/docs/how-tos/capabilities/), than hard-coded cluster ?

Reply:
BM cluster boot requires some configurations that are only available on build05.

Reply:
Used to be.   No longer the case AFAIK.
At least, re-provision build05 is one of the tasks in my todo list.

Reply:
updated.

Reply:
Looks like the newest change was to remove the build05 directly.  This is unexpected.  There is a doc link in the first comment of this thread which describes the expected change.

Reply:
@liangxia Thanks, updated. When you have time, could you help to review again?